plexapi/library.py

```diff
@@ -549,16 +580,25 @@ def search(self, title=None, sort=None, maxresults=999999, libtype=None, **kwarg
             args['sort'] = self._cleanSearchSort(sort)
         if libtype is not None:
             args['type'] = utils.searchType(libtype)
-        # iterate over the results
-        results, subresults = [], '_init'
+
+        results = []
+        subresults = []
         args['X-Plex-Container-Start'] = 0
```

Comment:
I just realized: With the introduction of `container_start` in #464 do we get a conflict here? This call specifies `X-Plex-Container-Start` to `0` but will this be overwritten by `PlexObject.fetchItems()`? And if not should we adjust this implementation to consider `container_start` as well to get the full flexibility?

Reply:
Nope, container_size and container_start in fetchItems is optional. Lets just reuse the entire thing